sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/details/key_constants.hpp

```diff
@@ -96,6 +96,19 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
   constexpr static const char A192KWValueValue[] = "A192KW";
   constexpr static const char A256KWValueValue[] = "A256KW";
 
+  /***************** Sign / Verify *********/
+  constexpr static const char RS256Value[] = "RS256";
+  constexpr static const char RS384Value[] = "RS384";
+  constexpr static const char RS512Value[] = "RS512";
+  constexpr static const char PS256Value[] = "PS256";
+  constexpr static const char PS384Value[] = "PS384";
+  constexpr static const char PS512Value[] = "PS512";
+  constexpr static const char ES256Value[] = "ES256";
+  constexpr static const char ES384Value[] = "ES384";
+  constexpr static const char ES512Value[] = "ES512";
+  constexpr static const char ES256KValue[] = "ES256K";
+  constexpr static const char DigestValue[] = "digest";
```

Comment:
What's this for?

Reply:
Serializing the Verify parameters to Json { value: ... , digest: .... ,  algo: ....}   it's just the constant for it